primitives/src/lib.rs

```diff
@@ -91,13 +91,8 @@ pub mod currency {
 	pub const CENT: Balance = DOLLAR / 100;
 	pub const MILLICENT: Balance = CENT / 1000;
 	/// The existential deposit.
-	#[allow(clippy::identity_op)]
-	#[cfg(feature = "integration-tests")]
 	pub const EXISTENTIAL_DEPOSIT: Balance = 1000;
```

Comment:
Need to make sure we all agree on this, or what could go wrong with very low `ED`?

The main reason for using this low ED is that some account distribution is very low and even smaller than ED, so node refused to add them.

Reply:
Yea we will have to clip those accounts from the distribution if they fall too low. We don't want to support accounts with such a low ED because that's like having a wallet that can't pay for TX fees. If 1 token is worth 1 cent, then 1000 / 1e18 is going to be like $0.0000000000000001

Reply:
Gotcha!